src/compiler/types.ts

```diff
@@ -361,17 +361,14 @@ namespace ts {
         // JSDoc nodes
         JSDocTypeExpression,
         JSDocNameReference,
-        // The * type
-        JSDocAllType,
-        // The ? type
-        JSDocUnknownType,
+        JSDocAllType, // The * type

```

Comment:
I'm wondering if these helpful comments should be leading `/** */` style comments, instead, so they actually show up in quick info by default... doesn't matter _too_ much, since as `//` comments these are visible if you hold `ctrl` while checking the quickinfo.